src/reducers/reducer1.js

```diff
@@ -0,0 +1,25 @@
+export const THE_ALERT = 'reducers1/THE_ALERT'
```

Comment:
reducers1 bad naming.

Reply:
I changed that on my latest Branch ,,,, That was just for setting up the Redux basic template